helm/flux-app/values.schema.json

```diff
@@ -197,7 +197,151 @@
           }
         }
       },
-      "additionalProperties": true
+      "additionalProperties": false
+    },
+    "volumes": {
+      "type": "object",
+      "required": [
+        "helmController",
+        "sourceController",
+        "kustomizeController",
+        "notificationController",
+        "imageAutomationController",
+        "imageReflectorController"
+      ],
+      "properties": {
+        "helmController": {
+          "type": "object",
+          "required": ["temp"],
+          "properties": {
+            "temp": {
+              "type": "object",
+              "required": ["size"],
+              "properties": {
+                "size": {
```

Comment:
Repeating elements can be factored out like this: https://github.com/giantswarm/efk-stack-app/blob/master/helm/efk-stack-app/charts/opendistro-es/values.schema.json#L2

Of course not urgent. Just FYI

Reply:
:bulb: ah yes thanks! I sensed this is possible but didn't bother to fix Will do now!